docs/build/program/apis/cloud.md

```diff
@@ -1332,3 +1332,262 @@ For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/
+
+**Parameters:**
+
+- `email` [(str)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): The email address the pending invite was sent to.
+
+**Returns:**
+
+- None.
+
+**Raises:**
+
+- `GRPCError`: This error is raised if no pending invite is associated with the provided email address.
+
+```python {class="line-numbers linkable-line-numbers"}
+await cloud.delete_organization_invite("youremail@email.com")
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/app/app_client/index.html#viam.app.app_client.AppClient.delete_organization_invite).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### ResendOrganizationInvite
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+Resend a pending organization invite email.
+
+**Parameters:**
+
+- `email` [(str)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): The email address associated with the invite.
+
+**Returns:**
+
+- None.
+
+**Raises:**
+
+- `GRPCError`: This error is raised if no pending invite is associated with the provided email address.
+
+```python {class="line-numbers linkable-line-numbers"}
+await cloud.resend_organization_invite("youremail@email.com")
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/app/app_client/index.html#viam.app.app_client.AppClient.resend_organization_invite).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### GetRoverRentalRobots
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+Return a list of rover rental robots within an {{< glossary_tooltip term_id="organization" text="organization" >}}.
+
+**Parameters:**
+
+- None.
+
+**Returns:**
+
+- [(List[viam.proto.app.RoverRentalRobot])](https://python.viam.dev/autoapi/viam/proto/app/index.html#viam.proto.app.RoverRentalRobot): The list of rover rental robots.
+
+```python {class="line-numbers linkable-line-numbers"}
+rental_robots = await cloud.get_rover_rental_robots()
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/app/app_client/index.html#viam.app.app_client.AppClient.get_rover_rental_robots).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### CheckPermissions
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+Check validity of a list of permissions.
+
+**Parameters:**
+
+- `permissions` [(List[viam.proto.app.AuthorizedPermissions])](https://python.viam.dev/autoapi/viam/proto/app/index.html#viam.proto.app.AuthorizedPermissions): A list of the resource permissions arguments to validate as authorized.
+  For example, `“read_organization”` or `“control_robot”`.
+  See [GitHub](https://github.com/viamrobotics/app/blob/main/auth/models/models.go#L190) for the full list of permissions.
+
+**Returns:**
+
+- [(List[viam.proto.app.AuthorizedPermissions])](https://python.viam.dev/autoapi/viam/proto/app/index.html#viam.proto.app.AuthorizedPermissions): The permissions argument, with invalid permissions filtered out.
```

Comment:
Hmm I'm realizing I'm not sure what this method does. @stuqdog ?
```suggestion
- [(List[viam.proto.app.AuthorizedPermissions])](https://python.viam.dev/autoapi/viam/proto/app/index.html#viam.proto.app.AuthorizedPermissions): The permissions argument, with invalid permissions filtered out.
For example, if you pass a location with its ID, followed by permissions `"read_fragment"` and `"write_fragment"`, and the location can read but not write fragments in the organization you are currently authenticated to, the method will return `["read_fragment"]`.
```
or is it
```suggestion
- [(List[viam.proto.app.AuthorizedPermissions])](https://python.viam.dev/autoapi/viam/proto/app/index.html#viam.proto.app.AuthorizedPermissions): The permissions argument, with invalid permissions filtered out.
For example, if you pass a location with its ID, followed by permissions `"read_fragment"` and `"write_fragment"`, and the organization you are currently authenticated to can read but not write fragments in the specified location, the method will return `["read_fragment"]`.
```
or none of the above? Sorry if these are bad examples--not sure how it works!

Reply:
I _believe_ it's the former. You pass some sort of ID (org, user, location, etc.) and then it returns which permissions that ID has access to. I suspect fleet knows more. @Kschappacher you were the last one to touch the `AuthorizedPermissions` type in app.proto (admittedly many months ago), can you confirm?

Reply:
Yupp that is correct. You can pass in any kind of resource id along with the permission you want to check and it will return subset of permissions the resource  actually has access to.